vault/auth.go

```diff
@@ -560,6 +549,26 @@ func (c *Core) newCredentialBackend(
 		return nil, fmt.Errorf("unknown backend type: %s", t)
 	}
 
+	// Set up conf to pass in plugin_name
+	conf := make(map[string]string, len(entry.Options)+1)
```

Comment:
This should probably be +2.

Reply:
What's the extra one for?

Reply:
Nevermind, I think I picked this up before uid was removed.  +1 is correct.

Reply:
This should be okay now that i removed the UID from the map